src/config.js

```diff
@@ -38,7 +39,13 @@ class Config {
     this.service = coalesce(options.service, platform.env('DD_SERVICE_NAME'), service, 'node')
     this.analytics = String(analytics) === 'true'
     this.runtimeId = coalesce(runtimeId, '')
-    this.tags = Object.assign({ 'runtime-id': this.runtimeId }, options.tags)
+    this.tags = Object.assign({}, options.tags, {
+      'runtime-id': this.runtimeId,
+      'language': 'javascript'
```

Comment:
for the `language` we should be specific because external services (i.e. `redis`) must not be tagged.